src/npm-fastui-bootstrap/src/footer.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react'
+import { components, useClassName } from 'fastui'
+
+export const Footer: FC<components.FooterProps> = (props) => {
+  const links = props.links.map((link) => {
+    link.mode = link.mode || 'footer'
+    return link
+  })
+  const extraSeparatorProp = useClassName(props, { el: 'extra-separator' })
+  const extraProp = useClassName(props, { el: 'extra' })
+  return (
+    <div className={useClassName(props, { el: 'contents' })}>
```

Comment:
footer doesn't seem to be placed correctly. It's height is taking up the rest of the page, rather than being of fixed height stuck to the bottom of the page.

Reply:
Hm yes, true.

I am quite unexperienced with CSS positioning things and tried around with a few options I found googling but couldn't get it to work as intended.

This example from bootstrap looked most promising but somehow doesn't work: https://getbootstrap.com/docs/5.0/examples/sticky-footer-navbar/

I may try again in the coming days but any hint would be welcome!